src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareDefault(a, b) {
```

Comment:
it would be better to move this function out of the `compareDefault`. Also,  `a,b` variable names should be replaced with the ones with more context